webdriver-spec.html

```diff
@@ -6778,7 +6801,8 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
  <li><p><a>Handle any user prompts</a>, and return its value if it is an <a>error</a>.
 
- <li><p>If the <var>name</var> matches a <a>cookie</a>’s <a>cookie name</a>
+ <li><p>If the <var>name</var>, from <var>url variables</var>,
```

Comment:
I would phrase this the other way around:

> If the _url variable_ _name_ matches a _cookie_’s _cookie name_

Reply:
Done